cmd/otk-make-fstab-stage/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/osbuild/images/pkg/osbuild"
+
+	"github.com/osbuild/images/internal/otkdisk"
+)
+
+type Input = otkdisk.Data
+
+func makeFstabStages(inp Input) (stages *osbuild.Stage, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = fmt.Errorf("cannot generate image prepare stages: %v", r)
+		}
+	}()
+
+	pt := inp.Const.Internal.PartitionTable
+	fstabStage := osbuild.NewFSTabStage(osbuild.NewFSTabStageOptions(pt))
```

Comment:
Let's make a note / issue to remove panics from these functions and have them return errors.  I know we've been saying it for a while now, but now that we're using them for these externals, I think we can at least prioritise the ones we want.

Reply:
Sure! i create a reminder for myself and will get to that in a followup.